openapi3filter/validate_response.go

```diff
@@ -159,11 +160,29 @@ func ValidateResponse(ctx context.Context, input *ResponseValidationInput) error
 
 	// Validate data with the schema.
 	if err := contentType.Schema.Value.VisitJSON(value, append(opts, openapi3.VisitAsResponse())...); err != nil {
+		schemaId := getSchemaIdentifier(contentType.Schema)
 		return &ResponseError{
 			Input:  input,
-			Reason: "response body doesn't match the schema",
+			Reason: fmt.Sprintf("response body doesn't match the schema %s", schemaId),
 			Err:    err,
+			// TODO: add an error message customizer function similar to SchemaError
```

Comment:
Open an issue instead of leaving that comment please.

Reply:
I opened issue #701